kalkulator/src/main/java/no/nav/folketrygdloven/kalkulator/guitjenester/refusjon/LagVurderRefusjonDto.java

```diff
@@ -145,7 +145,7 @@ private static Optional<BeregningRefusjonPeriodeDto> finnTidligereOverstyringFor
         if (Beløp.safeVerdi(årsbeløp) == null) {
             return Beløp.ZERO;
         }
-        return årsbeløp.divider(KonfigTjeneste.getMånederIÅr(), 0, RoundingMode.HALF_UP);
+        return årsbeløp.divider(KonfigTjeneste.getMånederIÅr(), 0, RoundingMode.DOWN);
```

Comment:
DOWN (kutter alle desimanler) eller HALF_DOWN? 
Hvilket utslag får dette for andre tilfelle enn i fagsystem? 
Blir man sittende med ufordelte 1-kroner med DOWN?

Reply:
Vi kutter allerede alle desimaler med scale 0. Alle saker som i dag runde opp går på trynet i neste steg fordi det ligger validering der som sjekker at refusjonskravet ikke er økt.

Reply:
Det er refusjonskravet man setter her, ikke grunnlag, eller fordelingen av det. Ideelt sett burde vi heller åpne for øre i aksjonspunktene

Reply:
Eg er enig med Jens Otto her. Kan dette føre til at vi får refusjonPrÅr som er 6 kroner lavere enn den eigentlege verdien? Dermed vil det stå 6 kr pr år i avkortetPrÅr til bruker

Reply:
Men i dag går det jo andre veien, at refusjonskrav på 1.6 = 2 kr?

Reply:
Men mulig det er bedre?

Reply:
Som sagt, den beste løsningen er uansett å heller fikse aksjonspunkt til å heller ta imot øre, så kanskje vi går rett for det istedenfor?

Reply:
Tolker det uansett dit hen at denne løsningen ikke blir riktig, så lukker pr

Reply:
Nå går jo dette på en Dto som kanskje er input til manuell fordeling (?)
Hvis man da bruker DOWN så kunne Dto få med et DIFF-felt på kroner som må fordeles på andeler